### ISSUE TITLE
Pipeline to test AzKS module

### ISSUE THREAD

**Issue by mkyc** [epiphany-platform/epiphany#1642]
**Is your feature request related to a problem? Please describe.**
Create pipeline to run tests on PR merge
**Describe the solution you'd like**
Pipeline should invoke tests already prepared in #1675
**Describe alternatives you've considered**
No.
**Additional context**
Check #1641 for information how t was done for AZBI module.

**Comment by mkyc** [comment]
Create separate task to implement tests.

**Comment by mkyc** [comment]
This goes back to TODO as was decided that testing itself will be done in #1674 and #1675

**Comment by atsikham** [comment]
Discussed with @plirglo how to create pipelines in Azure Devops. At this moment this task is blocked by https://github.com/epiphany-platform/m-azure-kubernetes-service/pull/2 and changes requested there.

**Comment by atsikham** [comment]
Please check: https://abb-epiphany.visualstudio.com/epiphany/_build?definitionId=101&_a=summary

**Comment by mkyc** [comment]
Both of them, in `Tests` section have one failing test: 

![Screenshot 2020-10-23 at 00 43 23](https://user-images.githubusercontent.com/28505782/96937687-03394900-14c9-11eb-82f0-011caf9b94aa.png)

Error is:

> Error: creating Managed Kubernetes Cluster "azks-module-tests-aks" (Resource Group "azks-module-tests-rg"): containerservice.ManagedClustersClient#CreateOrUpdate: Failure sending request: StatusCode=400 -- Original Error: Code="QuotaExceeded" Message="Provisioning of resource(s) for container service azks-module-tests-aks in resource group azks-module-tests-rg failed. Message: Operation could not be completed as it results in exceeding approved Total Regional Cores quota. Additional details - Deployment Model: Resource Manager, Location: GermanyWestCentral, Current Limit: 10, Current Usage: 7, Additional Required: 4, (Minimum) New Limit Required: 11. Submit a request for Quota increase at ...

My opinion is that it should make tests fail in PR but it's [very misleading right now](https://github.com/epiphany-platform/m-azure-kubernetes-service/pull/22/checks?check_run_id=1293371495).

**Comment by atsikham** [comment]
@mkyc fixed in new PR - https://github.com/epiphany-platform/m-azure-kubernetes-service/pull/24

![image.png](https://images.zenhubusercontent.com/5f29188b6ef7c40936a17a24/cea0430c-5bd3-47c5-9dbb-51590e2f7ef3)

**Comment by mkyc** [comment]
Verified runs.
